homeassistant/components/alexa_devices/quality_scale.yaml

```diff
@@ -51,16 +51,18 @@ rules:
   docs-known-limitations: todo
   docs-supported-devices: done
   docs-supported-functions: done
-  docs-troubleshooting: todo
+  docs-troubleshooting: done
   docs-use-cases: done
   dynamic-devices: todo
   entity-category: done
   entity-device-class: done
   entity-disabled-by-default: done
   entity-translations: done
-  exception-translations: todo
+  exception-translations: done
   icon-translations: done
-  reconfiguration-flow: todo
+  reconfiguration-flow:
+    status: exempt
+    comment: Amazon login name most likely won't change
```

Comment:
But you should still be able to trigger a reconfiguration, maybe because you changed your password and reauth didn't trigger yet, or maybe because you feel like it

Reply:
I don't get the real point honestly: if reauth didn't trigger is a bug. Should not be fixed here.

Anyway, moved to todo.